### ISSUE TITLE
RGB bulbs on v6 bridge?

### ISSUE THREAD

**Issue by b9chris** [Fantasmos/LimitlessLED-DevAPI#1]
Thanks for posting this.

I see the v6 full RGBW ("RGBW/WW/CW") bulbs are documented here, but, does anyone happen to have the documentation for the older RGB+W bulbs?

On the v6 bridge, the opening UDP bytes for a light command are always:
0x80, 0x00, 0x00, 0x00, 0x11, sess1, sess2, 0x00, seqNum, 0x00, 0x31, 0x00, 0x00

Where sess1 and 2 are the Session keys you got from opening a Session on the Bridge, and seqNum is just a rolling byte you keep incrementing with each send.

The total command is 22 bytes, and as you have in your docs, the full RGBW lights finish:
0x08, cmd, 0, 0, 0, val, group, chk

Where:
cmd is the ID of the command (like 0x01 for color)
val is the value of that command (like 0x80 for brightness)
group is which group in that category to talk to, 1-indexed (or pass 0 for all groups)
chk is the checksum of the rest of the command

So, the RGB+W older lights:
Can do Solid colors, but, no saturation
Can be switched between White mode, which is much brighter, and Color mode
No in-between and the white and color circuits can never power up simultaneously.

I have how to turn them off and on - they open like full RGBW lights but their final 8 bytes begin 0x07 where the full RGBWs open 0x08. For example switch on final 8 are:

0x07, 0x03, (isOn ? 0x01 : 0x02), 0, 0, 0, group, chk

Brightness:

0x07, 0x02, brightness, 0, 0, 0, group, chk

But I can't find the docs for the rest: How to switch between White and Color, and how to set the Color. Anyone have those?

**Comment by Supernova4422** [comment]
Is this the documentation you're looking for? https://github.com/Fantasmos/LimitlessLED-DevAPI/blob/master/RGBW.md

**Comment by b9chris** [comment]
I don't think so no, since none of those commands are in the format 0x07.... and I know for a fact that format is what turns these particular lights on over a v6 bridge. My guess is the documentation you're linking to there is for a different bridge or maybe different bulbs.
